apps/spruce/scripts/deploy/utils/git/git.test.ts

```diff
@@ -1,4 +1,3 @@
-import { execSync } from "child_process";
```

Comment:
I am not sure why this was only caught as a type error now... but `execSync` is unused in the file